opentelemetry-contrib/src/trace/exporter/jaeger_json.rs

```diff
@@ -49,7 +49,7 @@ impl<R: JaegerJsonRuntime> JaegerJsonExporter<R> {
     }
 
     /// Install the exporter using the internal provided runtime
-    pub fn install_batch(self) -> Tracer {
+    pub fn install_batch(self) -> (Tracer, TracerProvider) {
```

Comment:
this need to return the provider only. The caller then can create Tracer using the returned provider.

Reply:
Yeah, I thought about that, but my problem is the `InstrumentationScope` which is defined here and used to setup the tracer. Any advice on how to handle it? Or should I also remove it?

Reply:
You can remove it. The instrumentation scope is expected to capture the component emitting the traces, so whoever is emitting traces, they should call provider.tracer("component-name"), and use that tracer, rather than creating a tracer with hardcoded name here. (OpenTelemetry-OTLP was also doing this mistake - it was returning a tracer with hardcode named "otel-otlp", which defeats the purpose of InstrumentationScope, as every telemetry would be associated incorrectly with "otel-otlp", instead of reflecting the component)

Reply:
hey @cijothomas I did as asked, please have an other look again.